lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -50,8 +50,10 @@ namespace {
 // - the "Image" is used for icon in the context menu. Directly used std::string
 //   for these so the emplace_back() will choose the right constructor. With char[]
 //   it would convert that param to a bool and choose the wrong constructor.
+#ifdef WANT_QT_BUILD
 static constexpr char    kUSDLayerEditorItem[] = "USD Layer Editor";
 static constexpr char    kUSDLayerEditorLabel[] = "USD Layer Editor...";
+#endif
```

Comment:
When we don't have Qt, we don't build the Layer Editor so it should not be available in any menu.